.github/workflows/dart.yml

```diff
@@ -126,13 +130,17 @@ jobs:
       matrix:
         os: [macos-13, macos-14]
```

Comment:
Did you mean to add `macos-15` here as well? (And exclude the old version combinations, just like iOS)?

Because right now the new Xcode is added, but then excluded on macOS 13 and 14, which are all of the current options.